flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/expressions/ScalarFunctionsTest.scala

```diff
@@ -3971,7 +3971,7 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
   }
 
   @Test
-  def testFromUnixTimeInUTC(): Unit = {
+  def testFromUnixTime(): Unit = {
```

Comment:
should add FromUnixTime cases to `TemporalTypesTest`, there are many time zone env, like shanghai, DaylightSaving and etc.
Same to UnixTimestamp.